solution/index.js

```diff
@@ -0,0 +1,208 @@
+let tasks = window.localStorage.getItem("tasks") // all the tasks in the local storage
+let originTask; // saves and pass the <li> element that dragged in to other section  
+let altpressed = false; // global variable to check is alt is currently pressed
+let currentEl; // global variable that contains the <li> tag of the element that the mouse is currently over
```

Comment:
Do not use global variables. Having values that can be accessed and modified from different parts of a program is a clear recipe for bugs. There are better ways to both check whether `alt` is pressed, and to pass the hovered element to the keyboard event handler.